packages/native/types/base.d.ts

```diff
@@ -3,25 +3,21 @@
 
 import { ComponentPropsWithoutRef, ComponentType } from 'react'
 import * as RN from 'react-native'
-import { PropsOf } from '@emotion/core'
 
 type ReactNative = typeof RN
 
 export type ReactNativeStyle = ReturnType<ReactNative['StyleSheet']['flatten']>
 
-export interface ArrayInterpolation<MP> extends Array<Interpolation<MP>> {}
+export interface ArrayInterpolation<MergedProps>
+  extends Array<Interpolation<MergedProps>> {}
 
-// @ts-ignore
-export interface ObjectInterpolation
-  extends RN.ViewStyle,
-    RN.TextStyle,
-    RN.ImageStyle {}
+export type ObjectInterpolation = RN.ViewStyle | RN.TextStyle | RN.ImageStyle
 
 export interface FunctionInterpolation<MergedProps> {
   (mergedProps: MergedProps): Interpolation<MergedProps>
 }
 
-export type Interpolation<MergedProps = undefined> =
+export type Interpolation<MergedProps = unknown> =
```

Comment:
`unknown` is a more descriptive default for these `MergedProps` than `undefined`